src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -488,7 +488,11 @@ export class Connector implements IConnector {
      * * uses `favicon.ico` and the final url after redirects.
      */
     private async getFavicon(element?: AsyncHTMLElement) {
-        const href = element ? element.getAttribute('href') : '/favicon.ico';
+        if (!element || !element.getAttribute('href')) {
```

Comment:
@alrra Like http://www.example.com/favicon.ico returns 404. In the output of running sonar against http://www.example.com, we listed:
```
http://www.example.com: 1 issues
...
http://www.example.com/favicon.ico: 1 issues
...
```
But http://www.example.com/favicon.ico doesn't exist, it looks a little weird to still check its response.